### DISCUSSION TITLE
Health bars render above lightingN and lightingL in osu!mania

### DISCUSSION THREAD

**Discussion by joleskins** [ppy/osu#27906] [Ideas]
### Type

Cosmetic

### Bug description

Both argon and legacy health bars are above lighting in osu!mania, which looks off when the two elements overlap(like with the traditional HP position from stable)

This behaviour differs from stable where lighting would be above the health bar

After testing, this appears to be an issue on all skins including argon, legacy default, and multiple legacy skins imported from stable

### Screenshots or videos

Lazer - argon
![osu_2024-04-16_21-42-52](https://github.com/ppy/osu/assets/33073129/219cf25d-3b2d-4a7c-9289-8adbde52ed8b)

Lazer - imported skin from stable
![osu_2024-04-16_21-34-38](https://github.com/ppy/osu/assets/33073129/04e5d813-d9f5-4da2-b370-b47453ac2150)

Stable
![screenshot8509](https://github.com/ppy/osu/assets/33073129/02daa16b-73a0-4d1e-9c7b-8b06b6842c67)

### Version

2024.412.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/15001926/compressed-logs.zip)

**Comment by joleskins** [comment]
Tested with this skin in the screenshots, though testing with legacy default should suffice
[Clearium v1.5.zip](https://github.com/ppy/osu/files/15001971/Clearium.v1.5.zip)

**Comment by smoogipoo** [comment]
This is going to be extremely annoying to resolve, because lazer has a very strict separation between gameplay and the HUD. It may not be worth it.

**Comment by peppy** [comment]
Could be fixed by moving playfield level skin container below the playfield. Or having a new layer below.

https://github.com/ppy/osu/blob/4421ff975b330e832ee0405b9171dfeed2255577/osu.Game/Screens/Play/HUDOverlay.cs#L132-L134
